.github/workflows/testTKG.yml

```diff
@@ -0,0 +1,111 @@
+name: "Run TKG Tests"
+on:
+  issue_comment:
+    types: [created]
+jobs:
+  testTKG:
+    runs-on: ubuntu-latest
+    if: startsWith(github.event.comment.body, 'run tkg-test')
+    steps:
+      - uses: xt0rted/pull-request-comment-branch@v1
+        id: comment-branch
+
+      - uses: actions/checkout@v3
```

Comment:
These should be at the has commit version with a comment on the version tag

Reply:
Updated, thanks!